README.md

```diff
@@ -74,8 +74,7 @@ If you ever need help, feel free to ask around on our [Discord server](https://d
    As you make changes to the React application in `src`, those changes will be automatically reflected on the website locally.
 
 #### Running Tests
-1. go into `apps/antalmanac`
-2. `pnpm test`
+1. Run `pnpm test` in the root directory
```

Comment:
You can also run `pnpm -w test` from any subdirectory. But I don't really care about the readme lol. Figure it out or it's a skill issue 🤓